### PR TITLE
Add 0.167 release notes

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.167.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Hive Changes
+------------
+
+* Fix write operations for ``ViewFileSystem`` by using a relative location.
+* Remove support for the ``hive-cdh4`` and ``hive-hadoop1`` connectors which
+  support CDH 4 and Hadoop 1.x, respectively.
+* Remove the ``hive-cdh5`` connector as an alias for ``hive-hadoop2``.
+* Remove support for the legacy S3 block-based file system.
+* Add support for KMS-managed keys for S3 server-side encryption.
+
```
Comment (cawallin) [inline presto-docs/src/main/sphinx/release/release-0.167.rst]:
```
Cassandra Changes
--------------------------
* Add support for Cassandra 3.x by removing the deprecated Thrift interface used to connect to Cassandra. The following config options are now defunct and must be removed: ``cassandra.thrift-port``, ``cassandra.thrift-connection-factory-class``,  ``cassandra.transport-factory-options`` and ``cassandra.partitioner``.
```

Comment (cawallin) [inline presto-docs/src/main/sphinx/release/release-0.167.rst]: I can also submit a PR if you'd prefer

File: presto-docs/src/main/sphinx/release/release-0.167.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+Release 0.167
+=============
+
+General Changes
+---------------
+
+* Fix planning failure when a window function depends on the output of another window function.
+* Fix planning failure for certain aggregation with both ``DISTINCT`` and ``GROUP BY``.
+* Fix incorrect aggregation of operator summary statistics.
+* Fix a join operator issue that could cause joins that produce and filter many rows
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.167.rst]: maybe leave out the word "operator" in this one.   
```
Fix a issue in that could cause joins...
```

File: presto-docs/src/main/sphinx/release/release-0.167.rst [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  to monopolize worker threads, even after the query has finished.
+* Expand plan predicate pushdown capabilities involving implicitly coerced types.
+* Short-circuit inner and right join when right side is empty.
+* Optimize constant patterns for ``LIKE`` predicates that use an escape character.
+* Validate escape sequences in ``LIKE`` predicates per the SQL standard.
+* Reduce memory usage of :func:`min_by` and :func:`max_by`.
+* Add :func:`transform_keys`, :func:`transform_values` and :func:`zip_with` lambda functions.
+* Add :func:`levenshtein_distance` function.
+* Add JMX stat for the elapsed time of the longest currently active split.
+* Add JMX stats for compiler caches.
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-0.167.rst]: Do we want notes for new JMX stats?

File: presto-docs/src/main/sphinx/release/release-0.167.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.167.rst]: Thanks, I'll add that here after others are done reviewing

File: presto-docs/src/main/sphinx/release/release-0.167.rst (hunk 3 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.167.rst]: It's subjective, but these seem like something that could be useful to certain users. I don't think there is harm in putting them in the release notes.

**Comment by electrum** [comment]
I'm running the release now.

**Pull request closed without merging**